packages/account/src/providers/transaction-summary/operations.test.ts

```diff
@@ -243,134 +240,6 @@ describe('operations', () => {
     });
   });
 
-  describe('getTransferOperations', () => {
-    it('should ensure getTransferOperations return transfer operations from coin inputs', () => {
-      const expected: Operation = {
-        assetsSent: [
-          {
-            amount: bn('0x1'),
-            assetId: '0x0000000000000000000000000000000000000000000000000000000000000000',
```

Comment:
Were these tests just moved somewhere else?

Reply:
@Dhaiwat10 Sorry for not giving you a better context for this. I've updated the PR description.